source/hardware/controllers/denon_mc7000.rst

```diff
@@ -192,7 +192,8 @@ Layout and functions
 
 23. **Deck:** Selects which deck in the software is controlled by that
     hardware deck. The left deck can control Deck 1 or 3; the right deck
-    can control Deck 2 or 4.
+    can control Deck 2 or 4. Press and hold Shift and then press this
+    button to trigger the Platter LEDs mode.
 
 24. **Shift:** Press and hold this button to access secondary functions
```

Comment:
Good to know. What's the benefit using :hwlabel:`SHIFT`: over **Shift:** ?

Reply:
It enforces a consistent style when referring to labels on hardware. Some people wrote `Shift`, some wrote `SHIFT` or `[Shift]` or something like that. To the user, it becomes more readable as these are now emphasized by the stylesheet and they can now search for a button/knob/fader with that label on their controller even if the documentation does not have a schematic drawing with numbers.

Note that this only for *labeled* controls. Don't use it for Jog wheels when there isn't a label saying `JOG WHEEL` next to it.

Reply:
Thanks for explanation. Is there a guideline I can read through? Staying with the Jogwheel example it states **<SEARCH>** next to it using the Shift button. So would I need to bring that up a) separately b) just as comment for Jogwheel description or c) within the comment for Jogwheel like :hwlabel:`SEARCH`: ? So to extend it ... What about generally about shifted functions that are labelled on the controller ? How to define them in the manual? Cheers!

Reply:
The document is re-formatted. Unfortunately when building the manual locally with 'make html' i get error messages that hwlabel could not be used (WARNING: Unknown interpreted text role "hwlabel".) so to review the finished site is not always as good.

Also with this commit now all tests fail. Anything I can do about it?

Reply:
You need to need to install the dependencies (`pip install -r requirements.txt`) when you test locally. I suggest to do it in a [python virtual environment](https://docs.python.org/3/library/venv.html).

Reply:
After you push you can click on the deploy preview and check for errors.

Reply:
> Is there a guideline I can read through? Staying with the Jogwheel example it states **SEARCH** next to it using the Shift button. So would I need to bring that up a) separately b) just as comment for Jogwheel description or c) within the comment for Jogwheel like :hwlabel:`SEARCH`: ? So to extend it ... What about generally about shifted functions that are labelled on the controller ?

I think it makes sense to use

    :hwlabel:`SHIFT` + Jog Wheel

instead of:

    `hwlabel:`SEARCH`

I think that explicitly using SHIFT and the unshifted label makes it easier for the user to see what control you are talking about.

Reply:
> You need to need to install the dependencies (`pip install -r requirements.txt`) when you test locally. I suggest to do it in a [python virtual environment](https://docs.python.org/3/library/venv.html).

I already have installed dependencies but now did again. "Requirement already satisfied:"
Still it doesnt like hwlabel.

`$ pip install -r requirements.txt
Requirement already satisfied: sphinx in /home/tobias/.local/lib/python3.8/site-packages (from -r requirements.txt (line 1)) (3.2.1)
Requirement already satisfied: sphinx-intl in /home/tobias/.local/lib/python3.8/site-packages (from -r requirements.txt (line 2)) (2.0.1)
Requirement already satisfied: graphviz in /home/tobias/.local/lib/python3.8/site-packages (from -r requirements.txt (line 3)) (0.14.2)
Requirement already satisfied: markupsafe in /home/tobias/.local/lib/python3.8/site-packages (from -r requirements.txt (line 4)) (1.1.1)
Requirement already satisfied: transifex-client in /home/tobias/.local/lib/python3.8/site-packages (from -r requirements.txt (line 5)) (0.14.1)
Requirement already satisfied: sphinx-rtd-theme==0.4.3 in /home/tobias/.local/lib/python3.8/site-packages (from -r requirements.txt (line 6)) (0.4.3)
Requirement already satisfied: sphinx-multiversion==0.2.1 in /home/tobias/.local/lib/python3.8/site-packages (from -r requirements.txt (line 7)) (0.2.1)
Requirement already satisfied: Jinja2>=2.3 in /home/tobias/.local/lib/python3.8/site-packages (from sphinx->-r requirements.txt (line 1)) (2.11.2)
Requirement already satisfied: babel>=1.3 in /home/tobias/.local/lib/python3.8/site-packages (from sphinx->-r requirements.txt (line 1)) (2.8.0)
Requirement already satisfied: packaging in /home/tobias/.local/lib/python3.8/site-packages (from sphinx->-r requirements.txt (line 1)) (20.4)
Requirement already satisfied: sphinxcontrib-devhelp in /home/tobias/.local/lib/python3.8/site-packages (from sphinx->-r requirements.txt (line 1)) (1.0.2)
Requirement already satisfied: sphinxcontrib-qthelp in /home/tobias/.local/lib/python3.8/site-packages (from sphinx->-r requirements.txt (line 1)) (1.0.3)
Requirement already satisfied: docutils>=0.12 in /home/tobias/.local/lib/python3.8/site-packages (from sphinx->-r requirements.txt (line 1)) (0.16)
Requirement already satisfied: alabaster<0.8,>=0.7 in /home/tobias/.local/lib/python3.8/site-packages (from sphinx->-r requirements.txt (line 1)) (0.7.12)
Requirement already satisfied: sphinxcontrib-serializinghtml in /home/tobias/.local/lib/python3.8/site-packages (from sphinx->-r requirements.txt (line 1)) (1.1.4)
Requirement already satisfied: Pygments>=2.0 in /usr/lib/python3/dist-packages (from sphinx->-r requirements.txt (line 1)) (2.3.1)
Requirement already satisfied: sphinxcontrib-applehelp in /home/tobias/.local/lib/python3.8/site-packages (from sphinx->-r requirements.txt (line 1)) (1.0.2)
Requirement already satisfied: snowballstemmer>=1.1 in /home/tobias/.local/lib/python3.8/site-packages (from sphinx->-r requirements.txt (line 1)) (2.0.0)
Requirement already satisfied: setuptools in /usr/lib/python3/dist-packages (from sphinx->-r requirements.txt (line 1)) (45.2.0)
Requirement already satisfied: imagesize in /home/tobias/.local/lib/python3.8/site-packages (from sphinx->-r requirements.txt (line 1)) (1.2.0)
Requirement already satisfied: requests>=2.5.0 in /usr/lib/python3/dist-packages (from sphinx->-r requirements.txt (line 1)) (2.22.0)
Requirement already satisfied: sphinxcontrib-htmlhelp in /home/tobias/.local/lib/python3.8/site-packages (from sphinx->-r requirements.txt (line 1)) (1.0.3)
Requirement already satisfied: sphinxcontrib-jsmath in /home/tobias/.local/lib/python3.8/site-packages (from sphinx->-r requirements.txt (line 1)) (1.0.1)
Requirement already satisfied: click in /usr/lib/python3/dist-packages (from sphinx-intl->-r requirements.txt (line 2)) (7.0)
Requirement already satisfied: six<2.0.0 in /usr/lib/python3/dist-packages (from transifex-client->-r requirements.txt (line 5)) (1.14.0)
Requirement already satisfied: python-slugify<5.0.0 in /home/tobias/.local/lib/python3.8/site-packages (from transifex-client->-r requirements.txt (line 5)) (4.0.1)
Requirement already satisfied: gitpython<4.0.0 in /home/tobias/.local/lib/python3.8/site-packages (from transifex-client->-r requirements.txt (line 5)) (3.1.8)
Requirement already satisfied: urllib3<2.0.0,>=1.24.2 in /usr/lib/python3/dist-packages (from transifex-client->-r requirements.txt (line 5)) (1.25.8)
Requirement already satisfied: pytz>=2015.7 in /home/tobias/.local/lib/python3.8/site-packages (from babel>=1.3->sphinx->-r requirements.txt (line 1)) (2020.1)
Requirement already satisfied: pyparsing>=2.0.2 in /home/tobias/.local/lib/python3.8/site-packages (from packaging->sphinx->-r requirements.txt (line 1)) (2.4.7)
Requirement already satisfied: text-unidecode>=1.3 in /home/tobias/.local/lib/python3.8/site-packages (from python-slugify<5.0.0->transifex-client->-r requirements.txt (line 5)) (1.3)
Requirement already satisfied: gitdb<5,>=4.0.1 in /home/tobias/.local/lib/python3.8/site-packages (from gitpython<4.0.0->transifex-client->-r requirements.txt (line 5)) (4.0.5)
Requirement already satisfied: smmap<4,>=3.0.1 in /home/tobias/.local/lib/python3.8/site-packages (from gitdb<5,>=4.0.1->gitpython<4.0.0->transifex-client->-r requirements.txt (line 5)) (3.0.4)`

Reply:
Maybe your branch is too old. Run `git fetch upstream` and then `git rebase upstream/manual-2.3.x`. Then reinstall the dependencies.

PS: Please use 3 backticks for starting/ending code blocks. I edited your comment.